AutoTotem.cpp

```diff
@@ -0,0 +1,75 @@
+#include "AutoTotem.h"

```

Comment:
why is there a new file in the root folder

Reply:
fixed